src/components/common/Form/FormSelect.tsx

```
@@ -32,25 +34,31 @@ export function FormSelect({
-              required={required}
-              placeholder={placeholder}
-              value={field.value}
-              onChange={field.onChange}
-              allowEmpty={allowEmpty}
-            />
-          </FormControl>
-          <FormMessage />
-        </FormItem>
-      )}
+      render={({ field }) => {
+        const id = useId();
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)